src/view/sequence.rs

```diff
@@ -1,10 +1,32 @@
 use crate::event::MessageResult;
 use crate::id::Id;
-use crate::view::{Cx, View, ViewMarker};
+use crate::view::{Cx, TraitBound, ViewMarker};
 use crate::widget::{ChangeFlags, Pod, Widget};
 use crate::VecSplice;
 use std::any::Any;
 
+use super::view::{GenericView, WidgetBound};
+
+pub trait TraitPod {
+    type Pod: Element;
+
+    fn make_pod(w: Box<Self>) -> Self::Pod;
+}
+
+pub trait Element {
+    fn downcast_mut<T: 'static>(&mut self) -> Option<&mut T>;
+
+    fn mark(&mut self, flags: ChangeFlags) -> ChangeFlags;
+}
+
+impl TraitPod for WidgetBound {
+    type Pod = crate::widget::Pod;
+
+    fn make_pod(w: Box<Self>) -> Self::Pod {
```

Comment:
As i said on Zulip, i would like to avoid the explicit mention of `Box` in in `TraitPod `. Xilem#53 did'nt need it so maybe we can avoid it here too? 
But this is probably something we could change later without touching much code, as it only affects the element types.